examples/islands/chart.tsx

```diff
@@ -0,0 +1,9 @@
+import {
+  Chart as FreshChart,
+  type ChartProps,
+  type ChartType,
+} from "$fresh_charts/island.tsx";
+
+export default function Chart<Type extends ChartType>(props: ChartProps<Type>) {
+  return <FreshChart {...props} />;
+}
```

Comment:
```suggestion
export { Chart as default } from "$fresh_charts/island.tsx";
```
To be consistent with the example in README.